style.css

```diff
@@ -10,6 +10,9 @@ body {
 
 #aboutText {
 	margin-top: 2rem;
+	font-size: 14px;
+	width: 75%;
+	font-family: "Times New Roman";
```

Comment:
You should have a back-up font in case `"Times New Roman"` is not available. You can do this by using `font-family: "Times New Roman", serif`